test/Directory.Build.props

```diff
@@ -1,15 +1,25 @@
 <Project>
+  <Import Project="..\Directory.Build.props" />
+
   <PropertyGroup>
-    <_ImportParentFile>false</_ImportParentFile>
-    <_ImportParentFile Condition="$(MSBuildProjectName.StartsWith( 'Microsoft.AspNetCore.' ))">true</_ImportParentFile>
+    <DeveloperBuildTestTfms Condition=" '$(DeveloperBuildTestTfms)' == '' ">netcoreapp2.1</DeveloperBuildTestTfms>
```

Comment:
FYI this `Condition` is not in the Mvc version of this file. I added it to allow developers to regularly test two TFMs or otherwise override the `$(DeveloperBuild)` default.

Reply:
I don't need to test multiple Core TFMs but testing .NET Framework has caught a couple of problems. My AspNetCoreSettings.props file looks like
``` xml
<Project>
  <PropertyGroup>
    <DeveloperBuild>true</DeveloperBuild>
    <DeveloperBuildTestTfms>netcoreapp2.1</DeveloperBuildTestTfms>
    <DeveloperBuildTestTfms Condition=" '$(OS)' == 'Windows_NT' ">$(DeveloperBuildTestTfms);net461</DeveloperBuildTestTfms>
  </PropertyGroup>
</Project>
```

Without this `Condition`, my `$(DeveloperBuildTestTfms)` settings would be ignored. What is the problem with supporting this override?

That said, I could do something more like how we handle `$(DotNetRestoreSources)` in sources.repo i.e consume a new property that is not defined by default e.g.
``` xml
<DeveloperBuildTestTfms>$(AdditonalDeveloperBuildTestTargets);netcoreapp2.1</DeveloperBuildTestTfms>
```